.ci/manywheel/build_common.sh

```diff
@@ -111,12 +111,6 @@ case ${DESIRED_PYTHON} in
     ;;
 esac
 
-if [[ "$DESIRED_DEVTOOLSET" == *"cxx11-abi"* ]]; then
-    export _GLIBCXX_USE_CXX11_ABI=1
```

Comment:
I believe you should be setting ```export _GLIBCXX_USE_CXX11_ABI=1```  by default here

Reply:
You shouldn't actually need it [`CheckAbi.cmake`](https://github.com/pytorch/pytorch/blob/main/cmake/CheckAbi.cmake) should automatically set that for you.

https://github.com/pytorch/pytorch/blob/11d4438a5f1e9b6496fe0456ef5de82282d2ce24/cmake/CheckAbi.cmake#L7-L27

Reply:
You can see it in action here: https://github.com/pytorch/pytorch/actions/runs/13934198351/job/38998395184#step:14:730